### PR TITLE
Dom edit data access

### PR DESCRIPTION
**Pull request by terazus**
Contains:
- the edit data access pannel
- a mechanisms to warn the users when they leave the edit page while there are unsaved changes.
- a fix for dates in curator's unit tests.

### CODE REVIEW

File: src/views/CreateRecord/Editor.vue [hunk 1]
```diff
@@ -52,18 +52,24 @@
...
+            class="default"
             @click="panelData.show = true"
           >
             {{ panelData.name }}
           </v-btn>
+          <v-btn
+            class="ml-2 default"
+            :href="'#/' + $route.params.id"
+          >
+            Exist editing
```
Comment (knirirr) [inline src/views/CreateRecord/Editor.vue]: A typo has crept in here (exit -> exist).

**Review by knirirr — CHANGES_REQUESTED** [review]
I encountered a problem with the confirmation dialog when exiting editing; the first time it was triggered all worked as expected, but on subsequent occasions the exit editing button did nothing. 
I couldn't spot any other issues though.

File: src/views/CreateRecord/Editor.vue (hunk 1 above)
Comment (terazus) [reply src/views/CreateRecord/Editor.vue:71]: It's an issue with the link itself, not the code (try existing clicking on standard for instance, it works flawlessly).

**Comment by terazus** [comment]
It's an issue with the link itself, not the code (try existing clicking on standard for instance, it works flawlessly).

**Comment by knirirr** [comment]
> It's an issue with the link itself, not the code (try existing clicking on standard for instance, it works flawlessly).

Definitely not working for me, I can reproduce the problem.

**Comment by terazus** [comment]
> 
> 
> > It's an issue with the link itself, not the code (try existing clicking on standard for instance, it works flawlessly).
> 
> Definitely not working for me, I can reproduce the problem.

All buttons or only the exist one?

**Comment by knirirr** [comment]
Just the exit button, it seems.

**Review by knirirr — APPROVED** [review]

**Pull request merged**